### PR TITLE
kraken: periodically retry to load realtime data

### PR DESCRIPTION
**Pull request by kinnou02**

### CODE REVIEW

File: source/kraken/configuration.cpp [hunk 1]
```diff
@@ -55,6 +55,8 @@ po::options_description get_options_description(const boost::optional<std::strin
                                         "enable loading of realtime data")
         ("GENERAL.kirin_timeout", po::value<int>()->default_value(60000),
                                   "timeout in ms for loading realtime data from kirin")
+        ("GENERAL.kirin_retry_timeout", po::value<int>()->default_value(5*60),
```
Comment (antoine-de) [inline source/kraken/configuration.cpp]: a kirin_timeout in ms and a kirin_retry_timeout in s is a bit strange no ?

File: source/kraken/maintenance_worker.cpp [hunk 2]
```diff
@@ -242,7 +239,14 @@ void MaintenanceWorker::listen_rabbitmq(){
     LOG4CPLUS_INFO(logger, "start event loop");
     data_manager.get_data()->is_connected_to_rabbitmq = true;
     while(true){
+        auto now = pt::microsec_clock::universal_time();
+        //We don't want to try to load realtime data every second
+        if(now  > this->next_try_realtime_loading){
```
Comment (antoine-de) [inline source/kraken/maintenance_worker.cpp]: wrong spacing

File: source/kraken/maintenance_worker.cpp [hunk 3]
```diff
@@ -242,7 +239,14 @@ void MaintenanceWorker::listen_rabbitmq(){
...
+            this->next_try_realtime_loading = now + pt::seconds(conf.kirin_retry_timeout());
+            this->load_realtime();
```
Comment (antoine-de) [inline source/kraken/maintenance_worker.cpp:246]: i don't understand, we don't want to try if we already have some realtime no ?

Comment (kinnou02) [inline source/kraken/maintenance_worker.cpp:246]: `load_realtime` already check that, and others things, here we just don't want to try a call each second

File: source/kraken/configuration.cpp (hunk 1 above)
Comment (kinnou02) [inline source/kraken/configuration.cpp]: so, ms for everyone?

File: source/kraken/maintenance_worker.cpp (hunk 3 above)
Comment (antoine-de) [inline source/kraken/maintenance_worker.cpp:246]: ok

File: source/kraken/configuration.cpp (hunk 1 above)
Comment (antoine-de) [inline source/kraken/configuration.cpp]: as you wish

**Comment by kinnou02** [comment]
Done.

**Comment by kinnou02** [comment]
Jenkins, test this please.

**Pull request merged**